app/controllers/sessions_controller.rb

```diff
@@ -0,0 +1,21 @@
+# Manages the login functionality for users
+class SessionsController < ApplicationController
+
+  def new
+  end
+
+  def create
+    user = User.find_by_email(params[:email])
+    if user.try(:authenticate, params[:password])
+      session[:user_id] = user.id
+      redirect_to admin_url
```

Comment:
For slightly nicer behavior, set the redirect_url on the session, then redirect to that. Then the user ends up on the page they were trying to get o